src/ui/DynamicFacet/DynamicFacetValues/DynamicFacetValue.ts

```diff
@@ -87,15 +87,15 @@ export class DynamicFacetValue implements IDynamicFacetValue {
     const selectOrUnselect = !this.isSelected ? 'SelectValueWithResultCount' : 'UnselectValueWithResultCount';
     const resultCount = l('ResultCount', this.formattedCount);
 
-    return `${l(selectOrUnselect, this.valueCaption, resultCount)}`;
+    return `${l(selectOrUnselect, this.displayValue, resultCount)}`;
   }
 
   public get analyticsMeta(): IAnalyticsDynamicFacetMeta {
     return {
       ...this.facet.basicAnalyticsFacetState,
       value: this.value,
       valuePosition: this.position,
-      displayValue: this.displayValue ? this.displayValue : this.valueCaption,
+      displayValue: this.displayValue,
```

Comment:
Always using the `displayValue` for the analytics makes sense since the value caption does not exist in the index 👍